plugins/org.eclipse.n4js.generator.headless/src/org/eclipse/n4js/generator/headless/N4HeadlessCompiler.java

```diff
@@ -1001,7 +1022,7 @@ private ResourceSet createResourceSet() {
 	/**
 	 * Loads all resources in the given project and indexes them.
 	 *
-	 * FileSystemAccess has to be correctly configured, see {@link #configureFSA(IN4JSProject)} and {@link #resetFSA()}
+	 * FileSystemAccess has to be correctly configured, see {@link #configureFSA(IN4JSProject, Map)}
```

Comment:
I think that comment is not valid anymore. More like leftover from some old refactoring. 
Please verify and delete.

Reply:
removed.